src/rpc/blockchain.cpp

```diff
@@ -781,15 +781,13 @@ static RPCHelpMan getblockfrompeer()
         "getblockfrompeer",
         "\nAttempt to fetch block from a given peer.\n"
         "\nWe must have the header for this block, e.g. using submitheader.\n"
-        "\nReturns {} if a block-request was successfully scheduled\n",
+        "Subsequent calls for the same block and a new peer will cause the response from the previous peer to be ignored.\n"
+        "\nReturns an empty JSON object if the request was successfully scheduled.",
         {
-            {"blockhash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The block hash"},
-            {"nodeid", RPCArg::Type::NUM, RPCArg::Optional::NO, "The node ID (see getpeerinfo for node IDs)"},
+            {"block_hash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The block hash"},
+            {"peer_id", RPCArg::Type::NUM, RPCArg::Optional::NO, "The peer ID (see getpeerinfo for peer IDs)"},
         },
-        RPCResult{RPCResult::Type::OBJ, "", "",
-        {
-            {RPCResult::Type::STR, "warnings", /*optional=*/true, "any warnings"},
-        }},
+        RPCResult{RPCResult::Type::EMPTY, "", /* optional */ false, "", {}},
```

Comment:
Is the help now missing a brace?

previous reviews
```
Result:
{                        (json object)
  "warnings" : "str"     (string, optional) any warnings
}
```
now
```
Result:
     (json object)
}
```
maybe this would be nice
```
Result:
{
     (empty json object)
}
```

Reply:
2158b9f (nit, seems new since my last review) format for clang-tidy verification of this named arg in the RPCResult constructor
```suggestion
        RPCResult{RPCResult::Type::EMPTY, "", /*optional=*/ false, "", {}},

Reply:
I'll bring back the braces and fix the nit